hal/include/HAL/DriverStation.h

```diff
@@ -7,9 +7,7 @@
 
 #pragma once
 
-#include <stdint.h>
-
-#include <cstddef>
+#include <stddef.h>
```

Comment:
So for some reason this is failing our format tool, but completely removing it doesn't break anything, so we should probably just remove it completely.